pageserver/src/http/openapi_spec.yml

```diff
@@ -207,6 +207,115 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/tenant/{tenant_id}/timeline/{timeline_id}/do_gc:
```

Comment:
Maybe we can omit changes in this file entirely, since all of our new APIs don't work in prod, while this file is mainly intended to be used by `cloud` to get prod APIs.

Reply:
Makes sense, removed those changes.